src/virtualship/cli/commands.py

```diff
@@ -45,9 +47,76 @@ def init(path):
     "path",
     type=click.Path(exists=True, file_okay=False, dir_okay=True, readable=True),
 )
-def fetch(path):
-    """Entrypoint for the tool."""
-    raise NotImplementedError("Not implemented yet.")
+def fetch(expedition_dir: str | Path) -> None:
+    """Entrypoint for the tool to download data based on area of interest."""
+    if isinstance(expedition_dir, str):
+        expedition_dir = Path(expedition_dir)
```

Comment:
```suggestion
    expedition_dir = Path(expedition_dir)
```
no isinstance check needed. Also `expedition_dir` should be renamed to `path` (so click can function with it)